plugins/modules/postgresql_set.py

```diff
@@ -315,7 +315,7 @@ def param_set(cursor, module, name, value, context):
         if str(value).lower() == 'default':
             query = "ALTER SYSTEM SET %s = DEFAULT" % name
         else:
-            if isinstance(value, str) and ',' in value:
+            if isinstance(value, str) and ',' in value and not name.endswith('_command') and not name.endswith('_prefix'):
```

Comment:
```suggestion
            if isinstance(value, str) and ',' in value and not name.endswith(('_command', '_prefix')):
```
would this achieve the same logic?

Reply:
Yes, same logic but nicer code, thanks !!